Sao_Paulo_brazil_sugar_content_modeling.ipynb

```diff
@@ -0,0 +1,3619 @@
+{
```

Comment:
<p>"make sure that"? we know it is not converted to Date time. it is unnecessary to say that we are making sure. This is a necessary step for analysis.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gro-intelligence/api-client/pull/76/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>Done</p>